CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * Replace the component library with a new design pattern library [#822](https://github.com/hmrc/assets-frontend/pull/822) 
 * Add header component to the new design system 
 * Use macros for example and markup rendering in README.md
+* Added a 404 page pattern
```

Comment:
This needs a link to this PR. Look at previous entries for how to add one.

Reply:
✅